.travis.yml

```diff
@@ -41,4 +41,6 @@ install:
     - composer update $COMPOSER_FLAGS
     - ./vendor/bin/simple-phpunit install
 
-script: ./vendor/bin/simple-phpunit -v
+script:
+    - composer validate --strict --no-check-lock
```

Comment:
well, it breaks if the file is totally invalid, which would break the update.
But it does not break if the file has warnings making it unsuitable for publication on Packagist. And this is what happened in the past in some repos I maintain (stopping updates of the master branch in Packagist).